tests/integration/sync_test.go

```diff
@@ -8290,3 +8290,149 @@ func Test_Sync_Scoped_Plugins_Check_Conflicts(t *testing.T) {
 		})
 	}
 }
+
+// test scope:
+//
```

Comment:
Extra line?

Reply:
Not extra. We leave one in the middle before adding the scope.